app/src/components/v-workspace.vue

```diff
@@ -16,6 +16,7 @@
 				<v-workspace-tile
 					v-for="panel in panels"
 					:key="panel.id"
+					:loading="loading"
```

Comment:
Same thing as the other loading comment, I don't think this is the business logic of the v-workspace abstraction, but should rather be handled where the panels themselves construct the tile contents